src/Schema.ts

```diff
@@ -97,6 +99,11 @@ export interface ClickStep extends ClickAttributes, StepWithSelectors {
   duration?: number;
 }
 
+export interface HoverStep extends StepWithSelectors {
+  type: 'hover';
+  deviceType?: DeviceType;
```

Comment:
deviceType is not used?

Reply:
At the moment we don't use it. Should we remove this?

Reply:
Yes let’s remove it if it’s not used